.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,5 +1,5 @@
 connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
-connection.project.dir=
+connection.project.dir=../xtext-umbrella.git/releng/gradle-composite
```

Comment:
This requires update of the Oomph Setup, which will clone xtext-umbrella for all cases. The setup is prepared in this PR:
https://github.com/eclipse/xtext-eclipse/pull/118